test/shopify-cli/commands/connect_test.rb

```diff
@@ -10,34 +10,26 @@ def test_connect_asks_project_type_and_writes_yml_when_no_project_exists
         Resources::EnvFile.expects(:parse_external_env).returns({})
         ShopifyCli::PartnersAPI::Organizations.stubs(:fetch_with_app).returns(partners_api_response)
         CLI::UI::Prompt.expects(:ask).with(@context.message('core.connect.project_type_select')).returns('node')
-        CLI::UI::Prompt.expects(:ask).with(@context.message('core.connect.organization_select')).returns(101)
+        CLI::UI::Prompt.expects(:ask).with(@context.message('core.tasks.ensure_env.organization_select')).returns(101)
         CLI::UI::Prompt.expects(:ask).with(
-          @context.message('core.connect.development_store_select')
+          @context.message('core.tasks.ensure_env.development_store_select')
         ).returns('store.myshopify.com')
 
         Resources::EnvFile.any_instance.expects(:write)
         ShopifyCli::Project.expects(:write)
         run_cmd('connect')
       end
 
-      def test_connect_uses_default_values_for_env_file
-        ShopifyCli::Project.stubs(:has_current?).returns(false)
-        Resources::EnvFile.expects(:parse_external_env).returns({})
+      def test_connect_doesnt_write_yml_when_current_project_exists
         ShopifyCli::PartnersAPI::Organizations.stubs(:fetch_with_app).returns(partners_api_response)
-        CLI::UI::Prompt.expects(:ask).with(@context.message('core.connect.project_type_select')).returns('node')
-        CLI::UI::Prompt.expects(:ask).with(@context.message('core.connect.organization_select')).returns(101)
+        CLI::UI::Prompt.expects(:ask).with(@context.message('core.tasks.ensure_env.organization_select')).returns(101)
         CLI::UI::Prompt.expects(:ask).with(
-          @context.message('core.connect.development_store_select')
+          @context.message('core.tasks.ensure_env.development_store_select')
         ).returns('store.myshopify.com')
 
-        Resources::EnvFile.expects(:new).with(
-          api_key: 1235,
-          secret: 1234,
-          shop: "store.myshopify.com",
-          scopes: "write_products,write_customers,write_draft_orders",
-          extra: {}
-        ).returns(stub(:write))
-        ShopifyCli::Project.expects(:write)
+        CLI::UI::Prompt.expects(:ask).with(@context.message('core.connect.project_type_select')).never
+        Resources::EnvFile.any_instance.expects(:write)
```

Comment:
We should be expecting that this call is made `ShopifyCli::Tasks::EnsureEnv.call(@ctx, regenerate: true)` instead of `EnvFile.write` is called.